include/git2/tree.h

```diff
@@ -381,7 +381,7 @@ GIT_EXTERN(int) git_treebuilder_write(
  *
  * @see git_treebuilder_write
  *
- * @param id Pointer to store the OID of the newly written tree
+ * @param oid Pointer to store the OID of the newly written tree
```

Comment:
Yes, we decided a while back that the style is to use `id` as the name for object names, and "oid" is just a type. We did go through the API at one point to fix it all up, but it's quite probably they've snuck back in.